lib/calculation/stableswap.ak

```diff
@@ -15,7 +15,7 @@ pub fn swap_invariant(
     ),
     exchange_invariant(
       new_pool_gives,
-      new_pool_takes - takes_fees,
+      new_pool_takes - takes_fees - 1,
```

Comment:
@Quantumplation Adding this -1 seems to get everything to align between the [stableswap-scenarios](https://gist.github.com/Quantumplation/5f740287c6cc6b99dcd2c82b4d4bb46d) generated scenarios and the validator but I don't understand the exact mechanism

Reply:
I think because this ensures it rounds in favor of the protocol

Reply:
It's likely because it resolves the rounding in the favor of the protocol?